lib/src/layouts/yaru_compact_layout.dart

```diff
@@ -18,8 +21,8 @@ class YaruCompactLayout extends StatefulWidget {
     required this.length,
     required this.itemBuilder,
     required this.pageBuilder,
-    this.initialIndex = 0,
```

Comment:
this could be convenient to keep for the internally constructed controller. we could assert that only `initialIndex` or `controller` but not both are set. what do you think?

Reply:
Good point, it's probably more convenient to simply pass an argument than creating your own controller if all you need to do is set the initial index :+1: